server/DB/query/email-exist.js

```diff
@@ -0,0 +1,12 @@
+const connection = require('../config/connections');
+
+const emailExists= (email) => {
+    const sql = {
+        text: 'SELECT * FROM users WHERE email = $1',
```

Comment:
There is no need to select all the data from the table just to check if the user exists or no
returning the id maybe is more than enough

Reply:
I need to check the validate of the password with the same method, then pass data via cookie

that's why I select all of them